komga/src/main/kotlin/org/gotson/komga/interfaces/rest/SeriesController.kt

```diff
@@ -682,4 +684,25 @@ class SeriesController(
       .contentType(MediaType.parseMediaType("application/zip"))
       .body(streamingResponse)
   }
+
+  @DeleteMapping("v1/series/{seriesId}")
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  fun deleteSeries(
+    @PathVariable seriesId: String,
+    @AuthenticationPrincipal principal: KomgaPrincipal
+  ) {
+    seriesRepository.findByIdOrNull(seriesId)?.let { series ->
+      if (!principal.user.canAccessSeries(series)) throw ResponseStatusException(HttpStatus.FORBIDDEN)
+
+      val books = bookRepository.findAllBySeriesId(seriesId)
+      books.find { b -> !Files.isWritable(b.path) }?.let {
+        logger.warn { "Cannot delete ${it.path}: Permission denied" }
+        throw ResponseStatusException(HttpStatus.CONFLICT)
+      }
+
+      seriesLifecycle.deleteOne(series)
+      books.forEach { taskReceiver.deleteFile(it.url) }
```

Comment:
Deleting only the files contained in the folder would leave the folder itself, and the sidecars.